travvy/src/components/AttractionsList.vue

```diff
@@ -0,0 +1,32 @@
+<template>
```

Comment:
is it possible to rename this file so we dont get confused. just for consistency make it attraction page and attraction details page?

Reply:
yes, will do